src/fileattributefetch.cpp

```diff
@@ -31,6 +31,7 @@ FileAttributeFetchChannel::FileAttributeFetchChannel()
     urltime = 0;
     fahref = UNDEF;
     inbytes = 0;
+    e = API_EINTERNAL;
```

Comment:
I set it to `API_EINTERNAL` in the constructor because it should be set again before being used in a callback.